packages/core/src/utils/parseInvitation.ts

```diff
@@ -102,9 +105,33 @@ export const parseInvitationShortUrl = async (
   if (parsedUrl['oob']) {
     const outOfBandInvitation = OutOfBandInvitation.fromUrl(invitationUrl)
     return outOfBandInvitation
-  } else if (parsedUrl['c_i'] || parsedUrl['d_m']) {
+  } else if (parsedUrl['c_i']) {
     const invitation = ConnectionInvitationMessage.fromUrl(invitationUrl)
     return convertToNewInvitation(invitation)
+  }
+  // Legacy connectionless invitation
+  else if (parsedUrl['d_m']) {
+    const messageJson = JsonEncoder.fromBase64(parsedUrl['d_m'] as string)
```

Comment:
It uses base64url right? Maybe it does it properly under the hood, with `Buffer`, but it would be good to explicitly call it here.

Reply:
fromBase64 handles both base64 and base64url